python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -3542,6 +3539,13 @@ def keys(self):
                 self._key_columns[0], name=self.names[0]
             )
 
+    @property
+    def values_column_names(self) -> list[Hashable]:
```

Comment:
```suggestion
    def _values_column_names(self) -> list[Hashable]:
```

Does pandas have this public property? If not, should we just make it a private property?

Reply:
It doesn't, so I can make this private